config/core/configmaps/autoscaler.yaml

```diff
@@ -55,7 +72,9 @@ data:
     # NOTE: that this value does not affect actual number of concurrent requests
     #       the user container may receive, but only the average number of requests
     #       that the revision pods will receive.
-    container-concurrency-target-percentage: "70"
+    # This config will be deprecated, use target-utilization instead.
+    # Setting to "0" or remove it will disable this option.
+    container-concurrency-target-percentage: "0"
```

Comment:
We could drop this example and just make a note under `target-utilization` that  `container-concurrency-target-percentage` was the old map key that we still support